client/src/pages/Login/Login.tsx

```diff
@@ -52,7 +55,13 @@ export default function Login(): JSX.Element {
             </Grid>
             <LoginForm handleSubmit={handleSubmit} />
           </Box>
-          <Box p={1} alignSelf="center" />
+          <Box width="100%" className={classes.registerFooter}>
+            <Divider className={classes.divider} />
```

Comment:
Could we move the `width` property to the registerFooter class in the styles file?

Reply:
Sure!